Add Maltiverse integration section (#6257)
wazuh/wazuh-documentation: source/user-manual/manager/manual-integration.rst

   :align: center
   :width: 80%

Maltiverse
----------

> `Maltiverse <https://whatis.maltiverse.com/>`__ is an open and collaborative platform for indexing and searching Indicators of Compromise (IoCs). It works as a broker for Threat intelligence sources. Maltiverse aggregates information from more than a hundred different public, private and community sources. This integration identifies IoCs in Wazuh alerts via the Maltiverse API and generates new enriched ones inserting new fields, following the threat taxonomy of the ECS standard (Elastic Common Schema).

Reviewer: ```suggestion
`Maltiverse <https://whatis.maltiverse.com/>`__ is an open and collaborative platform for indexing and searching Indicators of Compromise (IoCs). It works as a broker for Threat intelligence sources. Maltiverse aggregates information from more than a hundred different public, private and community sources.

This integration identifies IoCs in Wazuh alerts via the Maltiverse API. It generates new alerts enriched with Maltiverse data. The Maltiverse data fields are based on the threat taxonomy of the ECS standard (Elastic Common Schema).
```

To set up this integration, do the following.

#. Get your API key from the `Maltiverse <https://www.maltiverse.com>`__ page.

#. Edit ``/var/ossec/etc/ossec.conf`` in the Wazuh server and include a configuration block such as the following. Replace ``MALTIVERSE_AUTH_TOKEN`` with your Maltiverse Authentication Token. The rule level filter is optional. You can remove it or set another level value for the integration.

   .. code-block:: xml
      :emphasize-lines: 3

      <integration>
         <name>custom-maltiverse</name>
         <hook_url>https://api.maltiverse.com</hook_url>
         <level>3</level>

>          <api_key>MALTIVERSE_AUTH_TOKEN</api_key>

Reviewer 2: Just as a comment, the other sections add a comment where values should be replaced, like 

```
<api_key>API_KEY</api_key> <!-- Replace with your VirusTotal API key -->
```

I don't think it's necessary but perhaps we should add it to be consistent with the rest of the page

   :align: center
   :width: 80%

Maltiverse
----------

`Maltiverse <https://whatis.maltiverse.com/>`__ is an open and collaborative platform for indexing and searching Indicators of Compromise (IoCs). It works as a broker for Threat intelligence sources. Maltiverse aggregates information from more than a hundred different public, private and community sources.

> This integration identifies IoCs in Wazuh alerts via the Maltiverse API and generates new enriched ones inserting new fields, following the threat taxonomy of the ECS standard (Elastic Common Schema).

Reviewer: ```suggestion
This integration identifies IoCs in Wazuh alerts via the Maltiverse API. It generates new alerts enriched with Maltiverse data. The Maltiverse data fields are based on the threat taxonomy of the ECS standard (Elastic Common Schema).
```
